### PR TITLE
Lots of content fixes, deletions, updates

### PR DESCRIPTION
**Pull request by mswilkison**
Fixes or removes outdated and/or incorrect information and links, including:
* Specify required node version in readme.
* Remove quiz functionality (I'm skeptical anyone uses this and it's distracting)
* Remove references and links to legacy staking post TIP-63
* Remove non-maintained events schedule section
* Add telegram and discourse forum links to navbar and footer
* Remove incorrect governance section
* Remove minters & guardians from home page (but leaves them in tbtc page)
* Some small typos, broken links, etc
* Link to defillama pool list rather than non-comprehensive manual list of tbtc and T pools

### CODE REVIEW

**Comment by mhluongo** [comment]
PR description plz ser

File: src/content/pages/earn/token-holder.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ tokenHolderInfo:
     next for Threshold. Legacy NU or KEEP tokens should first be upgraded to T.
   image: /images/token-holder.png
   buttons:
-    - label: Upgrade to T
-      url: https://dashboard.threshold.network/upgrade
+    - label: Delegate Vote
+      url: https://delegates.threshold.network
```
Comment (mhluongo) [inline src/content/pages/earn/token-holder.md:17]: This domain doesn't appear to resolve?

**Comment by mswilkison** [comment]
> PR description plz ser

Added, but mostly a restatement of commit descriptions.

File: src/hooks/useBalanceOfDAOTreasury.ts [hunk 2]
```diff
@@ -4,8 +4,8 @@ import { FixedNumber } from "ethers"
 import { CoingeckoID, exchangeAPI, formatUnits } from "../utils"
 import { useERC20TokensBalanceCall } from "./useERC20TokensBalanceCall"
 
-// https://etherscan.io/address/0x9f6e831c8f8939dc0c830c6e492e7cef4f9c2f5f
-const DAO_TREASURY_ADDRESS = "0x9f6e831c8f8939dc0c830c6e492e7cef4f9c2f5f"
+// https://etherscan.io/address/0x87F005317692D05BAA4193AB0c961c69e175f45f
```
Comment (mhluongo) [inline src/hooks/useBalanceOfDAOTreasury.ts:7]: Confirmed this is the DAO's Timelock address

Comment (mhluongo) [reply src/hooks/useBalanceOfDAOTreasury.ts:7]: It'd be really nice to include treasury details beyond just the timelock on the site somewhere. Outside the scope of this PR of course

**Review by mhluongo — COMMENTED** [review]
Couple quick nits. The deployed site looks good, but I'd like a +1 from @michalsmiarowski or @erdogan to merge

File: src/content/pages/earn/token-holder.md (hunk 1 above)
Comment (mswilkison) [reply src/content/pages/earn/token-holder.md:17]: asked for clarification on eta here: https://discord.com/channels/866378471868727316/870386636507185202/1178733196263825499

File: src/hooks/useBalanceOfDAOTreasury.ts (hunk 2 above)
Comment (mswilkison) [reply src/hooks/useBalanceOfDAOTreasury.ts:7]: > It'd be really nice to include treasury details beyond just the timelock on the site somewhere. Outside the scope of this PR of course

ya... query will just be more complex b/c need to capture LP and vote-escrowed positions, etc

Comment (mswilkison) [reply src/hooks/useBalanceOfDAOTreasury.ts:7]: I wonder if zapper or debank provide some kind of api or embed?

File: src/content/pages/ecosystem/index.md [hunk 3]
```diff
@@ -203,6 +175,11 @@ community:
     description: Ask questions and engage with our community.
     url: https://discord.gg/Threshold
     rightIcon: /images/external-arrow.svg
+  - leftIcon: /images/telegram-social.svg
+    title: Telegram
+    description: Ask questions and engage with our community.
```
Comment (jMyles) [inline src/content/pages/ecosystem/index.md:180]: Is it worth saying, "Ask questions and engage with our community.  Beware of scammers - we won't DM you first."

File: src/content/pages/governance/index.md [hunk 4]
```diff
@@ -99,7 +99,7 @@ guildsInfo:
           image: /images/162_2.jpg
       joinUrl: https://thresholdnetwork.notion.site/Integrations-Guild-4ff2e5fdea4442d7af19d27342ab8225
 migrationInfo:
-  title: Do you own KEEP or Nu and want to participate in Threshold Goverance?
+  title: Do you own KEEP or NU and want to participate in Threshold Goverance?
```
Comment (erdogan) [inline src/content/pages/governance/index.md:102]: typo: Gover**n**ance

**Review by erdogan — CHANGES_REQUESTED** [review]
@mswilkison all looks good to me. Two minor notes:  
- the delegates site is still offline
- small typo (Governance), added an inline comment for it

File: src/content/pages/governance/index.md (hunk 4 above)
Comment (mswilkison) [reply src/content/pages/governance/index.md]: Fixed in 1db32882c5f9cd96fbe557333516d0189bc7b918

**Review by erdogan — APPROVED** [review]

**Comment by mswilkison** [comment]
Force pushed to sign + verify commits. Ready for merge.

**Pull request merged**